src/Selector/MultipleSelector.tsx

```diff
@@ -122,27 +131,59 @@ const SelectSelector: React.FC<SelectorProps> = ({
     >
       {({ key, label, value, disabled: itemDisabled, className, style }) => {
         const mergedKey = key || value;
-
-        return (
+        const closable = key !== REST_TAG_KEY && !itemDisabled;
+        const onMouseDown = (event: React.MouseEvent) => {
+          event.preventDefault();
+          event.stopPropagation();
+        };
+        const closeTag = (event: React.MouseEvent) => {
+          if (event) event.stopPropagation();
+          onSelect(value, { selected: false });
+        };
+        const editTag = (event: React.MouseEvent) => {
```

Comment:
`editTag` seems not a Select logic that user should realize it in their own code logic. This prop should remove.